apps/web/src/app.tsx

```diff
@@ -164,8 +166,11 @@ function DesktopAppContents() {
 
   useEffect(() => {
     if (isListPaneVisible) {
-      navPane.current?.expand(1);
-      navPane.current?.reset(1);
+      if (navPane.current?.hasExpandedSize(1)) {
+        navPane.current?.expand(1);
+      } else {
+        navPane.current?.reset(1);
+      }
```

Comment:
I tried using `isCollapsed`, but it doesn't give enough information to decide whether to call `expand` or `reset`. That's why I need to use `hasExpandedSize`